Description: execute examples with Pyhton 3
 Update shebang to call Python 3 binary.
Author: Laszlo Boszormenyi (GCS) <gcs@debian.org>
Last-Update: 2020-03-24

---

--- protobuf-3.11.4.orig/examples/add_person.py
+++ protobuf-3.11.4/examples/add_person.py
@@ -1,4 +1,4 @@
-#! /usr/bin/env python
+#! /usr/bin/env python3
 
 # See README.txt for information and build instructions.
 
--- protobuf-3.11.4.orig/examples/list_people.py
+++ protobuf-3.11.4/examples/list_people.py
@@ -1,4 +1,4 @@
-#! /usr/bin/env python
+#! /usr/bin/env python3
 
 # See README.txt for information and build instructions.
 
